lib/select/index.js

```diff
@@ -188,6 +190,15 @@ export default class OCSelect extends React.Component {
     }
   };
 
+  getLableFromValue = value => {
+    const { options } = this.props;
+    const selectedOption = options.find(option => {
```

Comment:
With this one, can we rewrite the `find()` method with a good ol' fashioned `for` loop? IE doesn't support the `find()` method so we need to omit this, please.